swagger_to_sdk_config.json

```diff
@@ -30,7 +30,7 @@
       ]
     },
     "azure_mgmt_batch": {
-      "swagger": "arm-batch/2017-01-01/swagger/BatchManagement.json",
+      "swagger": "specification/batch/resource-manager/readme.md",
       "autorest_options": {
         "Namespace": "Azure::ARM::Batch",
```

Comment:
Note that autorest_options have changed, and you must use the new one:
- package-name and not packagename

This is not sensitive to case, so "Namespace" will still work.